learning/tour-of-beam/learning-content/java/schema-based-transforms/coder/description.md

```diff
@@ -0,0 +1,120 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Coder
+
+A Coder<T> defines how to encode and decode values of type T into byte streams.
```

Comment:
When Beam runners execute your pipeline, they often need to convert an object to and from a binary format. The Beam SDKs use objects called Coders to perform coding and decoding. 

In Beam SDK for Java, the base type `Coder` contains conversion methods. SDK also contains Coder subclasses that work with various standard Java types, such as Integer, Long, Double, StringUtf8, and more. You can find all the available Coder subclasses in the [Coder package](https://github.com/apache/beam/tree/master/sdks/java/core/src/main/java/org/apache/beam/sdk/coders).